src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -613,13 +614,14 @@ void InitializeTabularMonthly(EnergyPlusData &state)
             // #ifdef ITM_KEYCACHE
             //  Noel comment:  First time in this TabNum/ColNum loop, let's save the results
             //   of GetVariableKeyCountandType & GetVariableKeys.
-            std::string const &curVariMeter = UtilityRoutines::makeUPPER(ort->MonthlyFieldSetInput(FirstColumn + colNum - 1).variMeter);
+            std::string const curVariMeter = UtilityRoutines::makeUPPER(ort->MonthlyFieldSetInput(FirstColumn + colNum - 1).variMeter);
             // call the key count function but only need count during this pass
             int KeyCount = 0;
             GetVariableKeyCountandType(state, curVariMeter, KeyCount, TypeVar, AvgSumVar, StepTypeVar, UnitsVar);
             if (TypeVar == OutputProcessor::VariableType::NotFound) {
-                ShowWarningError(
-                    state, format("In Output:Table:Monthly '{}' invalid Variable or Meter Name '{}'", ort->MonthlyInput(TabNum).name, curVariMeter));
+                if (!ort->MonthlyInput(TabNum).isNamedMonthly) {
+                    ++state.dataOutRptTab->ErrCount1;
+                }
```

Comment:
Don't warn twice. This is my fault, from #8317

Reply:
We increment the count here.

Reply:
What purpose do you have in mind?

Note that that you also have a tracker on the MonthlyFieldSetInput (each "cell"), since keyCount is zero if it's an error.


---

Site note: I actually considered for a moment reporting in a different manner and grouping by table

```
   ** Warning ** In Output:Table:Monthly 'SPACE GAINS ANNUAL REPORT' invalid Variable or Meter Name(s):
   **   ~~~   ** - Variable X
   **   ~~~   ** - Variable Y
```

But this felt like it didn't follow 1) The existing reporting and 2) the general reporting pattern of eplusout.err